inc/class-shortcode-ui.php

```diff
@@ -35,13 +35,44 @@ private function setup_actions() {
 	public function register_shortcode_ui( $shortcode_tag, $args = array() ) {
 
 		$defaults = array(
-			'label'         => '',
-			'attrs'         => array(),
-			'listItemImage' => '',   // src or 'dashicons-' - used in insert list.
+			'label'             => '',
+			'attrs'             => array(),
+			'listItemImage'     => '',   // src or 'dashicons-' - used in insert list.
+			'inner_content'     => false,
 		);
 
 		$args = wp_parse_args( $args, $defaults );
-
+		
+		
+		// inner_content=true is a valid argument, but we want more detail
+		if ( is_bool( $args['inner_content'] ) && true === $args['inner_content'] ) {
+			$args['inner_content'] = array(
+				'label'            => esc_html__( 'Inner Content', 'shortcode-ui' ),
+				'description'      => '',
+				'placeholder'      => '',
+			);
+		}
+		
+		$num_attrs = count( $args['attrs'] );
```

Comment:
Can we have some mention that this is a backwards compatibility shim that we'll be removing in the next version?